vsts/scripts/dockerCleanup.sh

```diff
@@ -81,13 +80,9 @@ echo
 docker images
 
 echo
-echo "Cleanup: Run 'docker system prune': $DOCKER_SYSTEM_PRUNE"
```

Comment:
instead of removing it, we could just set `declare -r DOCKER_SYSTEM_PRUNE="${ORYX_DOCKER_SYSTEM_PRUNE:-true}"`

Reply:
true, but i didn't find any valid use case, do we use this variable during build?